src/msw/dcscreen.cpp

```diff
@@ -41,7 +41,7 @@ wxScreenDCImpl::wxScreenDCImpl( wxScreenDC *owner ) :
 void wxScreenDCImpl::DoGetSize(int *width, int *height) const
 {
     if ( width )
-        *width = ::GetSystemMetrics(SM_CXVIRTUALSCREEN);
+        *width = wxGetSystemMetrics(SM_CXVIRTUALSCREEN);
```

Comment:
Not sure if this is the right thing to do... Arguably, `wxScreenDC` should always use the primary monitor DPI and not the DPI of whichever monitor the application TLW is displayed on, as it's always associated with it (isn't it?).

This is not very important anyhow, of course, `wxScreenDC` is pretty useless with multiple monitors anyhow, but it might be better to just avoid changing it.

Reply:
Reverted